website/src/content/concepts/integrate-with-your-own-api.mdx

```diff
@@ -170,6 +201,8 @@ The Exchange token structure consist of **Registered Claim Names** and **Private
 - `iat`: This value represents the time at which the JWT was issued.
```

Comment:
```suggestion
- `iat`: **issued at** represents the time at which the JWT was issued.
```

Reply:
Updated in this commit: c8053312999955e37e23bc1c73a9fb8896af5314